link/tracing.go

```diff
@@ -87,6 +88,14 @@ type TracingOptions struct {
 	// AttachTraceFEntry/AttachTraceFExit/AttachModifyReturn or
 	// AttachTraceRawTp.
 	Program *ebpf.Program
+	// Program attach type. Can be one of:
```

Comment:
I think the current code also allows AttachLSMMac, etc. It'd be good to return an error.